### PR TITLE
What's different with PyCon India 2020 Devsprints

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Requesting @anistark for review.

File: content/blog/2020-announce-devsprints.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## What is a Devsprint?
+
+During **Devsprints**, mentors/contributors of popular OSS projects guide participants through the end-to-end process of an OSS contribution. Right from setting up the build environments, to developing and testing code, to submitting patches for bugs and features. If you are stuck anywhere, mentors will be available right on the stream to help you figure out issues and resolve them as best as they can! From past experiences, we’ve seen a lot of people start off contributing to FOSS through PyCon India Devsprints and later go on to become regular contributors to various projects across the web.
+
+## When and Where? Are Devsprints really free?
+
+Devsprints will be conducted online on 4th and 5th October 2020 on the Hopin.to conference platform. The [Devsprints tickets](https://in.pycon.org/2020/#ticket) are free of charge and limited in number as slots for the devsprint are limited. Please make sure to grab yours when they are released if you are looking to kickoff your FOSS journey right!
```
Comment (palnabarun) [inline content/blog/2020-announce-devsprints.md:17]: Tickets are gone now. I don't know if there is a point of that.

**Comment by sayanchowdhury** [comment]
This was discussed in a meeting before we published the Devsprints Tickets that we don't need this blog, rather we discussed that we would need a blog with the differences this year. @Sc0rpi0n101

**Review by palnabarun — CHANGES_REQUESTED** [review]
On a broad point, we had dicussed on the content team channel and meeting that DevSprints announcement posts are not required anymore. A mail was sent 2 weeks back to announce the same. We need one blog which just describes the DevSprint flow and what happens in a DevSprint.

File: content/blog/2020-announce-devsprints.md (hunk 1 above)
Comment (anistark) [reply content/blog/2020-announce-devsprints.md:17]: More tickets will come. right?

**Comment by anistark** [comment]
> This was discussed in a meeting before we published the Devsprints Tickets that we don't need this blog, rather we discussed that we would need a blog with the differences this year. @Sc0rpi0n101

In that case, let's modify this blog with "What's different at this year's PyCon India 2020 DevSprints" or something like that.

File: content/blog/2020-announce-devsprints.md (hunk 1 above)
Comment (palnabarun) [reply content/blog/2020-announce-devsprints.md:17]: We don't have a guarantee on that. :(

Comment (anistark) [reply content/blog/2020-announce-devsprints.md:17]: I believe we will have more. Sanchit said. We also announced at Flames event that there'll be more. And also promising the project maintainers while getting more projects.

Comment (palnabarun) [reply content/blog/2020-announce-devsprints.md:17]: All I am saying is we don't need to put speculation into the words of the blog. This thing needs to be communicated through social media.

File: content/blog/2020-announce-devsprints.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Title: git commit -s -m "Announing Devsprints 2020"
```
[the remark this answers is not in the document]
Comment (Sc0rpi0n101) [reply content/blog/2020-announce-devsprints.md:1]: It kinda looks good for a devsprint title.

File: content/blog/2020-announce-devsprints.md (hunk 1 above)
Comment (anistark) [reply content/blog/2020-announce-devsprints.md:17]: Makes sense. 👍🏼

**Comment by vipulgupta2048** [comment]
@sayanchowdhury @anistark @palnabarun modified the blog according to comments + also pitched a little workshops

**Comment by vipulgupta2048** [comment]
@palnabarun @sayanchowdhury @anistark  Lazy consensus to merge before 10 pm IST today for PPG

**Comment by anistark** [comment]
Looks good.

**Pull request merged**